src/components/PodcastPostLayout/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+
+type Props = {
+  title: string;
+  description: string;
+  markdown: string;
+};
+
+const PodcastPostLayout = ({ title, description, markdown }: Props): JSX.Element => {
+    const components = {
+      h2: ({ children }: { children: any }) => (
```

Comment:
Can we change this any?

https://www.typescriptlang.org/docs/handbook/jsx.html

Reply:
![image](https://github.com/mrruby/Ada/assets/103185922/369a411b-5fc5-483f-91a6-9a9e52731c28)

Can you suggest how I should write it? I don't think that's ok.

Reply:
![image](https://github.com/mrruby/Ada/assets/103185922/8582c9e0-d51c-40eb-9446-ef702155887c)

Reply:
```
  const components = {
    h2: ({ ...props }) => (
      <h2 className="font-bold md:text-adaSubtitle py-3" {...props} />
    ),
    h3: ({ ...props }) => <h3 className="font-bold py-3" {...props} />,
    h4: ({ ...props }) => <h4 className="font-bold py-3" {...props} />,
    ul: ({ ...props }) => <ul className="list-disc pl-6 pb-4" {...props} />,
    ol: ({ ...props }) => <ol className="list-decimal pl-6 pb-4" {...props} />,
    p: ({ ...props }) => <p className="pb-4" {...props} />,
  }
  ```
  
  How about now? Is this working?

Reply:
Yes, it's working !

Reply:
ufff